test/databases.test.js

```diff
@@ -1187,6 +1189,34 @@ const HasManyDissociator = require('../src/services/has-many-dissociator');
               connectionManager.closeConnection();
             }
           });
+
+          it('should handle numbers over MAX_SAFE_INTEGER even if there are fields that are not big int', async () => {
+            expect.assertions(2);
+            const { models, sequelizeOptions } = initializeSequelize();
+
+            // HACK: sequelize-fixtures does not support BigInt in json files,
+            //       so we need to update the clicks value manually
+            const counter = await models.counter.findByPk(10);
+            counter.clicks = BigInt('9013084467599484828'); // eslint-disable-line no-undef
```

Comment:
[MISC] I don't get why this `// eslint-disable-line no-undef` is needed here

Reply:
Otherwise I get a `'BigInt' is not defined` error by the linter... 
I used this trick as it was used in the same file but maybe you have a better idea! 🙏

Reply:
No no, I just checked the eslint doc and it wasn't clear at all why casting in bigint required this specific exception :)